public/lib/trusted-url.js

```diff
@@ -0,0 +1,8 @@
+import angular from 'angular';
+
+angular.module('wfTrustedUrl', [])
```

Comment:
this is because Angular (rightfully) doesn't trust injected external URLs in `ng-src`